zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/InterpreterResultMessage.java

```diff
@@ -16,18 +16,30 @@
  */
 package org.apache.zeppelin.interpreter;
 
+import org.apache.zeppelin.tabledata.ColumnDef;
+
 import java.io.Serializable;
+import java.util.List;
 
 /**
  * Interpreter result message
  */
 public class InterpreterResultMessage implements Serializable {
   InterpreterResult.Type type;
   String data;
+  MessageColumnTypes msgColumnTypes;
 
   public InterpreterResultMessage(InterpreterResult.Type type, String data) {
     this.type = type;
     this.data = data;
+    this.msgColumnTypes = new MessageColumnTypes();
```

Comment:
I think it's better to replace `this(type, data, new ArrayList<ColumnDef.TYPE>()`.

Reply:
Done